src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -263,6 +270,8 @@ public async Task RefreshAsync(CancellationToken cancellationToken)
                                 // Check if a change has been detected in the key-value registered for refresh
                                 if (change.ChangeType != KeyValueChangeType.None)
                                 {
+                                    logDebugBuilder.AppendLine(String.Format("{0}(key: \"{1}\", label: \"{2}\")", LoggingConstants.RefreshKeyValueChanged, change.Key, change.Label));

```

Comment:
Nit: we can use `'KeyName'` instead of `"KeyName"` to avoid escape characters.
Also, we can use [string interpolation](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated) instead of string.format everywhere.